src/components/avatar/avatar.js

```diff
@@ -25,23 +26,7 @@ const DEFAULT_SIZES = {
 }
 
 // --- Props ---
-const linkProps = pluckProps(
-  [
-    'href',
-    'rel',
-    'target',
-    'disabled',
-    'to',
-    'append',
-    'replace',
-    'activeClass',
-    'exact',
-    'exactActiveClass',
-    'prefetch',
-    'noPrefetch'
-  ],
-  BLinkProps
-)
+const linkProps = omit(BLinkProps, ['active', 'event', 'routerTag'])
```

Comment:
Is there any valid reason we omit `active` and `routerTag` here?

Reply:
Yeah.. they have no effect... and `routerTag` is really only when you are nesting router-links

Reply:
active styling is basically if the element is the active route... bootstrap's active styling can't bee seen here (active just adds the `active` class).

Reply:
And avatar doesn't have any special styling for the `active` class